models/excel_file.py

```diff
@@ -10,8 +10,8 @@
     Mapped,
     mapped_column,
 )
-from sqlalchemy.types import DateTime
 from sqlalchemy.schema import FetchedValue
+from sqlalchemy_utc.sqltypes import UtcDateTime
```

Comment:
This uses an extra package that implements a datetime type that matches the python datetimes better when it comes to time zones. This was annoying to deal with and I'll have to document it in the docs at some point.